common/package.json

```diff
@@ -26,7 +26,6 @@
     "@types/gtag.js": "^0.0.3",
     "@types/jest": "^27.0.2",
     "@types/lodash.debounce": "^4.0.6",
-    "@types/lodash.groupby": "^4.6.6",
```

Comment:
This is unrelated, but I spotted it here – we no longer use `lodash.groupby` anywhere (whereas we have other uses of `lodash.flattenBy`, so we don't need these types.